dns/py/runner.py

```diff
@@ -161,8 +162,6 @@ def _kubectl(self, stdin, *args):
     """
     cmdline = [self.args.kubectl_exec] + list(args)
     _log.debug('kubectl %s', cmdline)
-    if stdin:
-      _log.debug('kubectl stdin\n%s', add_prefix('in  | ', stdin))
```

Comment:
I removed this logging as it was causing more harm than good. We're logging stdin here, which can be anything, e.g. a tar archive...